src/pages/LoginPage.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser, faKey } from '@fortawesome/free-solid-svg-icons';
+
+import '../assets/styles/pages/LoginPage.css';
+
+function LoginPage(){
+    const [emailErrorVisibility, setEmailErrorVisibility] = useState('hidden');
+    const [passwordErrorVisibility, setPasswordErrorVisibility] = useState('hidden');
+
+    const [emailErrorMessage, setEmailErrorMessage] = useState('');
+    const [passwordErrorMessage, setPasswordErrorMessage] = useState('');
+
+    const [currentEmail, setCurrentEmail] = useState('');
+    const [currentPassword, setCurrentPassword] = useState('');
```

Comment:
It's a best practice to use objects and create one hook to handle changes, errors, etc. (**Use easy code**)

Reply:
Hello Leonel, thank you so much, now I'm using only one hook with an object for state